app/furniture/marketplace/delivery_area_component.rb

```diff
@@ -0,0 +1,41 @@
+class Marketplace
+  class DeliveryAreaComponent < ApplicationComponent
+    attr_accessor :delivery_area
+    delegate :label, to: :delivery_area
+
+    def initialize(delivery_area:, data: {}, classes: "")
+      super(data: data, classes: classes)
+
+      self.delivery_area = delivery_area
+    end
+
+    delegate :price, to: :delivery_area
+
+    def edit_button
+      return unless edit_button?
+
+      ButtonComponent.new label: "#{t("icons.edit")} #{t("edit.link_to")}",
```

Comment:
I think this would look better using `Buttons::SecondaryComponent` instead (it's new, @anaulin just made them and it's glorious!)

Reply:
fixed in dcc959cca01abf4e3155a8da8c80dc0142efc48b

<img width="1048" alt="Screenshot 2023-04-12 at 9 41 00 PM" src="https://user-images.githubusercontent.com/16140873/231655413-b9281772-fa87-47ce-8425-2c91405f8ced.png">